pandas/tests/series/methods/test_isin.py

```diff
@@ -234,3 +234,17 @@ def test_isin_filtering_with_mixed_object_types(data, is_in):
     expected = Series([True, False])
 
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "data",
+    [([1, 2, 3]), ([1.0, 2.0, 3.0])],
```

Comment:
looks like you've got some extra brackets (I'm surprised none of the linters caught this)
```suggestion
    [[1, 2, 3], [1.0, 2.0, 3.0]],
```

Reply:
Good catch! Thanks!